golang/go-cloud-run-hello-world/test_content.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# retries, interval,keyword may be passed as arguments
+# retries: -r
+# interval (in seconds): -i
+# keyword: -k
+# url: -u
+
+## Defaults
+keyword="Chocolate Chip"
+retries=10
+interval=5
+url="http://localhost:3000"
+
+while getopts r:i:k:u: option
+do
+    case "${option}"
+    in
+        r) retries=$OPTARG;;
+        i) interval=$OPTARG;;
+        k) keyword=$OPTARG;;
+        u) url=$OPTARG;
+    esac
+done
+
+echo "START CONTENT TEST"
+echo "retries: "$retries
+echo "interval: "$interval
+echo "keyword: "$keyword
+echo "url: "$url
+
+if [[ -z "$url" ]]; then
+  echo "Failed: URL not specified."
+  exit 1
+fi
```

Comment:
I added this stanza because during development of the `cloudbuild.yaml` I had an error where the URL was an empty value and the test process went through all retries for no reason.